app/js/reducers/fetchOrdersReducer.js

```diff
@@ -8,9 +10,23 @@ const initialState = {
   },
 };
 
+const filter = (from, sortBy, value) => {
+  const savedOrders = JSON.parse(localStorage.getItem('ORDERS')) || from;
+  switch (value) {
+    case 'all':
+      return savedOrders;
+    default:
+      if (from.length === 0) {
+        return savedOrders.filter(order => order[sortBy] === value);
```

Comment:
Some concerns on this line of code 
```
 if (from.length === 0) {
  ....
}
```
Let's assume this is the initial result:
```
[{ ... type: 'drugorder' ... }, { ... type: 'drugorder' ... }, { ... type: 'testorder' ... }, { ... type: 'xxxorder' ... }, ]
```
After filtering out all the `drugorder`, this is what `from` would be like
```
[{ ... type: 'testorder' ... }, { ... type: 'xxxorder' ... }, ]
```
And then I filter out all the `xxxorder`, `from` would become:
```
[{ ... type: 'testorder' ... }]
```
Lets say I want to get back all the `drugorders`, do you think I would be able to do that? don't you think this is breaking the functionality?

Reply:
hmm, I'll address this